libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -276,6 +277,12 @@ void _usage(void)
     printf("\tmodule support:\n");
     printf("\t                   --module-directory %s\n", AP_MODULE_DEFAULT_DIRECTORY);
     printf("\t                   -M %s\n", AP_MODULE_DEFAULT_DIRECTORY);
+#ifdef MQTT_ENABLED
+    printf("\tmqtt options:\n");
+    printf("\t                   --mqtt_server tcp:\\192.168.1.x:xxxx\n");
```

Comment:
Is the local IP address fixed? I think that it is better to use your own network address.